app/routes/__frontend/monthlychallenges/oct-2023.jsx

```diff
@@ -0,0 +1,483 @@
+			</p>
+
+			<p>
+				<strong>Contributors</strong>
+			</p>
+			<ul>
+				<li>
+					Sign up on the{' '}
+					<a href="https://hacktoberfest.com/">
+						Hacktoberfest official website
+					</a>
+					.
+				</li>
+				<li>Contact your mentor if you request one.</li>
+				<li>
+					Check out repositories with the "hacktoberfest" topic and issues on
+					GitHub with the "hacktoberfest" label.
+				</li>
+			</ul>
+
+			<p>
+				<strong>Maintainers</strong>
+			</p>
+			<p>
+				Make sure you include "hacktoberfest" as a label in your project's
+				topics section (found in the "About" section on your repository). That's
+				how you opt-in to Hacktoberfest.
+			</p>
+
+			<h3>Weekly async check-ins</h3>
+			<p>
+				Let's start the week with a check-in. What are your goals for
+				Hacktoberfest for the week? What support do you need? Do you plan on
+				spending time in the <code>#hacktoberfest-co-working-room</code>,
+				talking to your mentor, or answering questions as a maintainer?
+			</p>
+			<p>
+				Every Friday of the month, we will drop a check-in for you to share your
+				progress, ask questions, and find help and support.
+			</p>
+			<p>
+				We will drop the check-ins in the <code>#hacktoberfest</code> channel on
+				Slack.
+			</p>
+
+			<h3>How do I share my progress?</h3>
+			<p>
+				Share your progress in the <code>#hacktoberfest</code> channel in Slack
+				so we can celebrate with you! And if you need support, we'll be there to
+				help you, too.
+			</p>
+			<p>
+				While no other platform is imposed, sharing on social media for more
+				reach can be a good idea, but only if you are comfortable doing so. You
+				can share on X (formerly Twitter) using — or not — the hashtag{' '}
+				<code>#VCHI</code>, a personal blog, a post on{' '}
+				<a href="https://dev.to/">DEV.to</a>, you get the idea!
+			</p>
+			<p>
+				Sharing every time you add some work is a good idea. Small progress is
+				still progress. There is no need to write a detailed blog post. A
+				140-character post on X can be enough. For example, "
+				<em>Today, I submitted my first PR for Hacktoberfest. #VCHI</em>"
+			</p>
+
+			<h3>What if I need help?</h3>
+			<p>
+				You can ask questions and ask for ideas and help in the{' '}
+				<code>#hacktoberfest</code> channel in Slack or join the{' '}
+				<code>#hacktoberfest-co-working-room</code>. Asking for help is part of
+				the process!
+			</p>
+
+			<p>And remember, we're always here to help! 💙</p>
+
+			<h3>Completing the Challenge!</h3>
+			<p>
+				To complete the challenge, you need to complete four meaningful
+				contributions during October. It can be creating a PR, writing a good
+				issue, creating content such as a blog post or video about contributing
+				to a project, helping to answer questions or triaging issues, meeting
+				with your mentee, and so on. Share your progress in the{' '}
+				<code>#hacktoberfest</code> channel on Slack.
```

Comment:
As per dicussion on Slack, I changed the wordings here to suit the "four meaningful contributions during October".
Previously:

> Any kind of participation in Hacktoberfest counts as you complete the VC October challenge.